stats/stats_test.go

```diff
@@ -720,7 +720,50 @@ func TestNaN(t *testing.T) {
 	}
 }
 
+func TestBucketLookUp(t *testing.T) {
+	var tests = []struct {
+		input float64 // input
+		start float64 // start
+		end   float64 // end
+
+	}{
+		{input: 999, start: 900, end: 1000},
+		{input: 999.99, start: 900, end: 1000},
+		{input: 1000.0004, start: 1000, end: 2000},
```

Comment:
great, but please put 1000 itself too

Reply:
I added it because of 0.0001 value in the record. When I put 1000, it becomes 999,999

Reply:
yes because the intervals are open on the end https://github.com/istio/fortio/blob/master/stats/stats.go#L121
so let's verify - replace the 1000.0004 which is same as next line by 1000 and update the start/end

Reply:
Okay then, for instance; for 1000, start will be 900 and end will be 1000.